src/components/app/UrlManager.js

```diff
@@ -36,9 +49,67 @@ type Props = ConnectedProps<OwnProps, StateProps, DispatchProps>;
 
 /**
  * This component manages the interaction with the window.history browser API.
+ *
+ * There are two different profile loading paths currently:
```

Comment:
Added some documentation about the different profile loading paths. Let me know what you think.